### ISSUE TITLE
Bug: Github Actions not compatible with local testing

### ISSUE THREAD

**Issue by pncnmnp** [TommasU/scrivener#33]
Running the build prepared locally on Github Actions is generating an `AssertionError: Model None does not exist.`

On a closer inspection of the `punctuator` module, we can observe that:

```python
        model_file = self.model_exists(model_file)
        assert model_file, 'Model %s does not exist.' % model_file

        self.model_file = model_file
```
This is the line generating the error. As the assert statement is getting a `None` value from `model_file`, it breaks the test case. Also, this indicates that the `model_file` is `None`. Even after changing the file path from relative to absolute (a couple of variations of their implementations) this problem persists.

**Comment by pncnmnp** [comment]
For more detail on the changes made, see:
* 490365791b97b91fe76de136d6640f5a053014d2
* 676483b2dae3c37832615f70ec9e8f7cda7ee273
* 47dedc960ccfcef11a8b2aec2e1c13b318d4b51b
* 640bc01cb69b3af2b3dd017cd9499cce696f4fc4

3 tests are passing. However, the 4 tests dependent on the `punctuator` module are failing.

**Comment by pncnmnp** [comment]
The issue was fixed after analyzing the issues in the `punctuator` library. We found this issue - https://github.com/chrisspen/punctuator2/issues/6

The issue was caused due to lack of file on that file path. I have defined `conftest.py` file with the initial configuration required for continuous testing.

As of the latest build: https://github.com/TommasU/scrivener/actions/runs/1417718397

This has been fixed. Closing the issue for now.
